bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -369,6 +372,27 @@ async def list_nominations(
             lines.append(line)
         return lines
 
+    @staticmethod
+    async def maybe_relay_update(api: NominationAPI, bot: Bot, nominee_id: int, update: str) -> None:
+        """
+        Checks for an active nomination thread for the user, if one is found relay the given update there.
+
+        This is used to relay new nominations and nomination updates to active vote channels.
+        """
+        nominations = await api.get_nominations(user_id=nominee_id, active=True, reviewed=True)
+
+        found_thread_id = None
+
+        for nomination in nominations:
+            if nomination.thread_id:
+                found_thread_id = nomination.thread_id
+
+        if found_thread_id:
+            log.debug(f"Found thread ID for {nominee_id} nomination, relaying new context.")
+            thread = await get_or_fetch_channel(bot, found_thread_id)
+
+            await thread.send(update)
+
```

Comment:
If we added an `api.get_active_nomination(user_id) -> Nomination | None` function, I think this could be made a lot cleaner. I'd been thinking of suggesting that anyway as it'd be useful in quite a few other places as well.
```suggestion
        nomination = await api.get_active_nomination(nominee_id)
        
        if nomination and nomination.thread_id:
            log.debug(f"Found thread ID for {nominee_id} nomination, relaying new context.")
            thread = await get_or_fetch_channel(bot, nomination.thread_id)

            await thread.send(update)

```